website/package.json

```diff
@@ -19,13 +17,16 @@
     "write-heading-ids": "docusaurus write-heading-ids",
     "typecheck": "tsc"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
We have no actual runtime -- everything's a devDep. We could consider the App things to be "prod deps" if we want and I can refactor, but the current setup in main has no consistency...